package.json

```diff
@@ -154,5 +162,23 @@
   "bugs": {
     "url": "https://github.com/indico/indico/issues"
   },
-  "homepage": "https://github.com/indico/indico#readme"
+  "homepage": "https://github.com/indico/indico#readme",
+  "jest": {
+    "moduleNameMapper": {
+      "^.+\\.(s?css)$": "identity-obj-proxy",
+      "^indico/modules/([\\w]+)/(.*)(.jsx?|)$": "<rootDir>/indico/modules/$1/client/js/$2$3",
```

Comment:
Does the `(.jsx?|)` part make sense? No matter what you have there, it will either go in `$3` or `$2`